mlflow/server/js/src/experiment-tracking/components/MetricsPlotPanel.tsx

```diff
@@ -60,6 +89,9 @@ type OwnMetricsPlotPanelProps = {
   navigate: NavigateFunction;
   runDisplayNames: string[];
   containsInfinities: boolean;
+  // Additional props for new chart system
+  runNames?: string[];
+  runInfos?: any[];
```

Comment:
tip: you can probably use `RunInfoEntity` type from `mlflow/server/js/src/experiment-tracking/types.ts`